### DISCUSSION TITLE
Eurós egységárak és visszajáró Forinttól eltérő fizetőeszközben

### DISCUSSION THREAD

**Discussion by AttilaPletyak** [nav-gov-hu/eRECEIPT#11] [Ötletek]
A pénztárgépek szűk szegmensét érinti (nemzetközi utasoknak kiállított bizonylatok), hogy a kereskedő adhasson meg egységárakat is Euróban, illetve adhasson visszajárót is Euróban. Ez a helyzet például a reptéren, vagy tranzit útvonalak mellett, ahol a vásárló már nem szeretne Forintot visszakapni, hanem valamilyen valutát, pl Eurót. Az Online Számla rendszerben van erre lehetőség, az új szabályozásban ki kellene térni ennek kezelésére is.

**Comment by BatariZ** [comment]
Az időközben megjelent rendelet (A nemzetgazdasági miniszter 8/2025. (III. 31.) NGM rendelete az e-pénztárgépek forgalmazásáról, üzemeltetéséről, valamint az e-pénztárgépek és az e-nyugta kiállításának követelményeiről), illetve az aktualizált Fejlesztői dokumentáció tartalmazza a válaszokat a feltett kérdésekre.
